frameworks/kafka/src/main/dist/svc.yml

```diff
@@ -85,14 +85,30 @@ pods:
         {{/TASKCFG_ALL_SECURITY_KERBEROS_DEBUG}}
       {{/TASKCFG_ALL_SECURITY_KERBEROS_ENABLED}}
 
+          # Getting needed env variables from bootstrap
+          ./bootstrap -resolve=false 2> bootstrap_stderr
```

Comment:
Since all we are interested in is the environment variables, we could use the following:
```
export MESOS_CONTAINER_IP=$( ./bootstrap --get-task-ip )
export LIB_PROCESS_IP=$MESOS_CONTAINER_IP
```

Reply:
OK, wasn't aware of this option.

Reply:
These changes should now be merged into `master`. See #2510.